datadog/scripts/tilt.py

```diff
@@ -284,13 +284,20 @@ def resource_type(self):
         return "uiresource"
 
     def _test_status(self) -> bool:
-        status = self.properties["status"]["disableStatus"]["state"]
-        return status == "Enabled"
+        try:
+            status = self.properties["status"]["disableStatus"]["state"]
+            return status == "Enabled"
+        except KeyError:
+            return False
 
     def _push_state(self, enable: bool = True):
-        tilt_xable(enable=enable, port=self.port, **self.toggle_args)
+        tilt_xable(
+            enable=enable,
+            port=self.port,
+            **self.toggle_args,
```

Comment:
This change was unnecessary:

```suggestion
            **self.toggle_args
```

Reply:
Doesn't the trailing comma adhere to [Black formatting](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas)?

Reply:
If you remove the trailing comma, the expression can fit on a single line.